### PR TITLE
Implement file create

### PR DESCRIPTION
**Pull request by ztplz**
Prior art:
`go:  func Create(name string) (*File, error)`
`rust: tokio::fs::File::create<P>(path: P) -> CreateFuture<P>`

### CODE REVIEW

File: js/files_test.ts [hunk 1]
```diff
@@ -18,3 +18,21 @@ test(async function filesCopyToStdout() {
   assertEqual(bytesWritten, fileSize);
   console.log("bytes written", bytesWritten);
 });
+
+test(async function createFileSuccess() {
+  const path = "test.txt";
+  let f = await deno.create(path);
```
Comment (kevinkassimo) [inline js/files_test.ts]: Could you create the test file in a temporary directory? Like using a path given by `deno.makeTempDir`

Comment (ztplz) [reply js/files_test.ts]: Sure.

Comment (ztplz) [reply js/files_test.ts]: Sure.

File: src/ops.rs [hunk 2]
```diff
@@ -558,6 +559,41 @@ fn op_mkdir(
   })
 }
 
+fn op_create(
```
Comment (ry) [inline src/ops.rs]: Reuse op_open

**Comment by ztplz** [comment]
@ry  I think permission mode should be completed before restart it. Next I will try to work on permission  design.

**Comment by hayd** [comment]
If `create` can be defined as `writeFileSync('touched', new Uint8Array(), 0o666);` does it need to be defined by deno?

**Comment by ry** [comment]
@ztplz I'm going to close this one as it's getting dated - if you come back to this project please open a new PR and link to this one. Thanks!

**Pull request closed without merging**